### ISSUE TITLE
How to activate the "refresh" button in the Transfer menu from distance?

### ISSUE THREAD

**Issue by elbre** [keeps/roda#1537]
Good day,
in process of our ingest workflow, we are transferring the sip packages directly into the transferred resources folder. These SIPs will not show in RODA website and are not registered by the API transferred resources list request as well unless I will use the "refresh" button inside the RODA website.
Is there any way how "refresh" menu Ingest/Transfer from a distance (for example by API)?

**Comment by luis100** [comment]
Yes, use the REST API method for this.

https://demo.roda-community.org/api-docs/

GET /v1/transfers/reindex

You can reindex the whole transfer or a relative folder within transfer.

Please note that you must use preemptive basic authentication to login with a RODA user that has permissions to execute that method

https://demo.roda-community.org/api-docs/

Example:
```bash
curl -X GET "https://demo.roda-community.org/api/v1/transfers/reindex?transferred_resource_path=test&acceptFormat=json" -H "accept: application/json" -u admin:roda
```

**Comment by luis100** [comment]
@elbre heads up, we are about to release 3.4.0 with some fixes relative to the transfer reindex API #1540
